docs/api/en/renderers/WebGLRenderer.html

```diff
@@ -168,8 +168,8 @@ <h3>[property:Object extensions]</h3>
 		</ul>
 		</p>
 
-		<h3>[property:number outputEncoding]</h3>
-		<p>Defines the output encoding of the renderer. Default is [page:Textures THREE.LinearEncoding].</p>
+		<h3>[property:number outputColorSpace]</h3>
+		<p>Defines the output colorspace of the renderer. Default is [page:Textures THREE.LinearEncoding].</p>
```

Comment:
Typo: "color space", not "colorspace".

Reply:
I'll note that — as far as I know — both spellings are correct. But I'd like to be consistent and "color space" is what I've stuck to throughout the documentation so far.